# HG changeset patch
# User Neil <nyamatongwe@gmail.com>
# Date 1491552139 -36000
# Node ID c9dcde8a91d3c5d9f28ba3a7cead581ff880e818
# Parent  84161436cfbc309da3b30e6fe5bc69c0a9331cfa
Drop "virtual" from methods that do not need to be virtual.

diff -r 84161436cfbc -r c9dcde8a91d3 src/Decoration.h
--- a/src/Decoration.h	Fri Apr 07 17:28:01 2017 +1000
+++ b/src/Decoration.h	Fri Apr 07 18:02:19 2017 +1000
@@ -66,10 +66,10 @@
 	int Start(int indicator, int position);
 	int End(int indicator, int position);
 
-	virtual bool ClickNotified() const {
+	bool ClickNotified() const {
 		return clickNotified;
 	}
-	virtual void SetClickNotified(bool notified) {
+	void SetClickNotified(bool notified) {
 		clickNotified = notified;
 	}
 };
